sdk/azidentity/interactive_browser_credential.go

```diff
@@ -34,8 +34,9 @@ type InteractiveBrowserCredentialOptions struct {
 	// The localhost port for the local server that will be used to redirect back.
 	// By default, a random port number will be selected.
 	Port int
-	// Disables use of PKCE during authorization.
+	// Deprecated: Disables use of PKCE during authorization.
 	// The default is false.
+	// TODO: remove
 	DisablePKCE bool
```

Comment:
Do we need to leave this note since we're in preview?

Reply:
We could remove it now, just we had a minor bump last week (I probably overthink it).

Reply:
I mean I understand where you're coming from...but I feel like leaving it like this during preview we might accidentally forget to remove it at some point and then cause ourselves problems? If we leave it like this then I guess I'd say let's open an issue to track removing it?

Reply:
Removed